src/Bridge/Symfony/Bundle/Resources/views/Graphiql/index.html.twig

```diff
@@ -11,6 +11,10 @@
 <body>
 <div id="graphiql">Loading...</div>
 
+<script>
+    var entrypoint = '{{ entrypoint }}';
```

Comment:
Is it really a good idea to use an unnamespaced global variable like this?

Reply:
I don't think it's really harmful, but how could it be done differently?

Reply:
https://github.com/api-platform/core/blob/9aba73225cbeca44bce34c8250fe0c2494cbfb6f/src/Bridge/Symfony/Bundle/Resources/views/SwaggerUi/index.html.twig#L13-L14

Reply:
In your case it'd just be `type="text/plain"`, of course.

Reply:
Done thanks.